engines/director/archive.h

```diff
@@ -80,7 +81,9 @@ class Archive {
 	Common::SeekableReadStream *_stream;
 	typedef Common::HashMap<uint16, Resource> ResourceMap;
 	typedef Common::HashMap<uint32, ResourceMap> TypeMap;
+	typedef Common::HashMap<uint32, uint32> MovieMap;
```

Comment:
I'd call this MovieChunkMap to make it clear this maps to chunk IDs, not movies.